pppd/main.c

```diff
@@ -1863,9 +1863,9 @@ update_script_environment(void)
     struct userenv *uep;
 
     for (uep = userenv_list; uep != NULL; uep = uep->ue_next) {
-	int i;
+	size_t i;
```

Comment:
For a case like this where the variable is a loop index rather than the size or length of something, I'd prefer plain long or unsigned long rather than size_t. Same comment applies in a few other places.